nebula/ui/components/VPNAlert.qml

```diff
@@ -13,26 +13,26 @@ Rectangle {
     id: alertBox
 
     // Defines what type of alert this is
-    property var alertType: stateNames.info
+    property string alertType: stateNames.info
     // Contains all accepted values for alertType
     readonly property var alertTypes: stateNames
     // Determines if the toast should fit into the layout or put itself on the top
-    property var isLayout: false
+    property bool isLayout: false
     // Callback when the action was pressed,
     // (does nothing if  no action text is defined)
     property var onActionPressed:function(){}
     // Callback when the close button was pressed
     property var onClosePressed:function(){}
     // Text of the toast
-    property var alertText: "Some Alert"
+    property string alertText: "Some Alert"
```

Comment:
For properties like this, I think we should remove the default assigned value ("Some Alert") and utilize the `required` attribute so we don't ever accidentally end up with an unintended value.